### PR TITLE
Add a blog to Zazu's documentation site. [skip ci]

### PR DESCRIPTION
**Pull request by bayleedev**
This allows us to talk about videos, walkthroughs, releases and new plugins.

This PR includes:

* A blog link at the top
* Pagination on the blog
* Correct breadcrumbs
* A post about the introduction video
   * This includes the video
   * And a blog-ified version of the content
* Showing the blog in search results

@twang2218, do you have time to review this?

### CODE REVIEW

File: docs/_includes/navigation.html [hunk 1]
```diff
@@ -5,6 +5,8 @@
     <a class="c-navigation__item {% if page.url == '/documentation/' %}is-active{% endif %}" href="/documentation/">Docs</a>
     <a class="c-navigation__item {% if page.url == '/plugins/' %}is-active{% endif %}" href="/plugins/">Plugins</a>
     <a class="c-navigation__item {% if page.url == '/themes/' %}is-active{% endif %}" href="/themes/">Themes</a>
+    <a class="c-navigation__item {% if page.url == '/themes/' %}is-active{% endif %}" href="/themes/">Themes</a>
```
Comment (amjith) [inline docs/_includes/navigation.html]: Why do you have 'themes' listed twice?

File: docs/_posts/2017-03-07-video-introduction-to-zazu.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+To install Zazu please visit the [website and download][download] the
+appropriate binary for your operating system. Zazu is cross-platform compatible
+and we currently supply binaries for Windows, Mac and Ubuntu.
+
+## Using Zazu
+
+Once it is successfully installed, you can launch Zazu. You’ll notice the
+feather icon in the menu bar of your operating system. You can now open Zazu
+sing the shortcut key `Alt-Space`. Launching an application is as simple as
```
Comment (amjith) [inline docs/_posts/2017-03-07-video-introduction-to-zazu.md]: ~sing~ using

File: docs/_posts/2017-03-07-video-introduction-to-zazu.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+typing a few letters from the name of the program. For example typing `chr`
+will show `Google Chrome` in the results.
+
+## Default Plugins
+
+As I mentioned earlier, Zazu ships with a rich set of plugins that go beyond the
```
Comment (amjith) [inline docs/_posts/2017-03-07-video-introduction-to-zazu.md]: Lose `As I mentioned earlier`.

File: docs/_posts/2017-03-07-video-introduction-to-zazu.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+Another useful plugin that ships by default is the [clipboard
+manager][clipboard] which can be launched using `Cmd-Shift-V`. You can search
+through this list of items you’ve copied earlier in the day. It can also show
+you a handy preview of the images you have copied.
+
+## Installing Plugins
+
+But the real power of Zazu lies with the plugins contributed by users to satisfy
+their specific needs. You can browse through a list of available plugins on the
+[plugins page][plugins page]or you can look at them straight from Zazu by typing
```
Comment (amjith) [inline docs/_posts/2017-03-07-video-introduction-to-zazu.md]: Add space before `or you....`

File: docs/_includes/navigation.html (hunk 1 above)
Comment (bayleedev) [reply docs/_includes/navigation.html]: I really like themes, geez.

**Pull request merged**